### ISSUE TITLE
Add LINQBridge to NuGet

### ISSUE THREAD

**Issue by atifaziz** [atifaziz/LINQBridge#22]
```
It would be great to get this on NuGet as many of my projects' only remaining 
/lib/ folder reference is this DLL.
```

Original issue reported on code.google.com by `*dan...@wildbit.com` on 23 May 2011 at 4:53

**Comment by atifaziz** [comment]
Original comment by `azizatif` on 23 May 2011 at 9:32
- Changed state: **Accepted**
- Added labels: **Component-Scripts**, **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by atifaziz** [comment]
Original comment by `azizatif` on 23 May 2011 at 9:32
- Changed title: **Add LINQBridge to NuGet**

**Comment by atifaziz** [comment]
```
any news on this?
```

Original comment by `mcintyre...@gmail.com` on 20 Mar 2012 at 11:21

**Comment by atifaziz** [comment]
Original comment by `azizatif` on 16 Apr 2012 at 9:23

**Comment by atifaziz** [comment]
Original comment by `azizatif` on 17 Apr 2012 at 8:12
- Added labels: **Milestone-Release1.3**

**Comment by atifaziz** [comment]
```
This issue was closed by revision 8c4aad8f7699.
```

Original comment by `azizatif` on 18 Apr 2012 at 4:14
- Changed state: **Fixed**
